ouroboros-consensus-test/test-consensus/Test/Consensus/BlockchainTime/Simple.hs

```diff
@@ -288,8 +289,10 @@ prop_delayNoClockShift =
                    (slotLengthFromMillisec 100)
                    (secondsToNominalDiffTime 20)
                    5
-      assertEqual "slots" slots [SlotNo n | n <- [0..4]]
+      pure $ slots === [SlotNo n | n <- [0..4]]
 
+-- | Note that that under load, the returned list could be missing certain slots
```

Comment:
Do we need to prepend this with a `TODO` or `FIXME`?

Reply:
I added a link to the tech debt issue: https://github.com/input-output-hk/ouroboros-network/compare/3fb8d691f2ed562484b673f9f9c55a4fb74e3486..f65d9093e8fa2e1284bf3a0e4bfc6b9aef2f70f3